torch/distributed/rpc/options.py

```diff
@@ -81,17 +83,15 @@ def __init__(
         init_method: str = rpc_contants.DEFAULT_INIT_METHOD,
         device_maps: Optional[Dict[str, Dict[DeviceType, DeviceType]]] = None,
         devices: Optional[List[DeviceType]] = None,
-        _transports: List = None,
-        _channels: List = None,
+        _transports: Optional[List] = None,
+        _channels: Optional[List] = None,
```

Comment:
Would I be correct if these two lines are the only non-formatting change in this file?

Reply:
Yep! For context, this is part of a project to migrate Pytorch from Mypy to Pyre. Pytorch requires `Optional` types to be explicitly specified while Mypy will automatically infer it as `Optional` if it takes a value of `None`.